### ISSUE TITLE
R-Link 2 24 pin diagram

### ISSUE THREAD

**Issue by larskka** [cedricp/ddt4all#1038]
Hy hope someone can help me. have a scenic iv from 2020 where i want to install reverse camera. anyone can help with a diagram to r-link 2 with 24 pin have tryed 18 cam+ 19 cam - 20 cam + suply but i dont work. thr camera an wire work when i connect to another schreen, so i think it is the wiring there is wrong...

**Comment by larskka** [comment]
![stik](https://github.com/cedricp/ddt4all/assets/161332104/0c78d5c8-09b4-4bd5-87ac-f5c7f8c3f26a)
this is what i have tried drawn as looking inside the connection on r-link

**Comment by AndyIreland** [comment]
if its a car without a reverse camera and you have put it on as an add on/upgrade I should imagine you will have to enable rear reverse camera in the navigation section with DDT4ALL on your rlink2

**Comment by larskka** [comment]
Yes its is new and activated with ddt4all. it has now the camera menu, but says defet camera or missing signal. so i think i dident use the right pins? i have bought at adapter  but it is 32 pin and the radio has 24 pin so i found the connections on the picture but i dont work

**Comment by larskka** [comment]
it is 6 volt. if i connect camera wit power from extern cable there is still no video signal on r link . but there is picture if i connect the video cable to a old tv...

**Comment by AndyIreland** [comment]
strange , should work then you would think. - so you are using the +6v and -6v pins and the CVBs  + and - and putting car into reverse gear and no pic ? - where did you get the 24pin wiring detail diagram from. 

here is wiring for rlink1 but I dont know if it the same for rlink2:
![image](https://github.com/cedricp/ddt4all/assets/49172217/214e8b74-beb3-4e97-bc61-9adf413319c7)

![image](https://github.com/cedricp/ddt4all/assets/49172217/92491077-3800-42ca-bd8d-3bb26d8ad80b)
![image](https://github.com/cedricp/ddt4all/assets/49172217/a87ce102-a2d7-4d8e-8203-8ed32bb5efa2)
![image](https://github.com/cedricp/ddt4all/assets/49172217/8e5bc79c-f16d-4c5a-b0e2-d7b484e9cf82)

**Comment by larskka** [comment]
I think I saw it on this site and it fit fine with the pins that were not in use. so just like you have shown here but - for the camera supply I have taken out the rear lights. But it is not only the power that has problems, because I have tried to supply the camera with external power, so the video signal is also wrong. When the car is put into reverse gear, a black screen appears and it reports a camera error or lack of signal

**Comment by AndyIreland** [comment]
yes, it could possibly be different wiring for the rlink2 vs  the rlink1 24 pin connector then

**Comment by larskka** [comment]
My thought is that mabye there i a different wirring in r-link2 ?

**Comment by AndyIreland** [comment]
I found this - pin 21 could it be on the 32pin cable?
https://github.com/cedricp/ddt4all/issues/696

![image](https://github.com/cedricp/ddt4all/assets/49172217/232b9b6a-b2f1-4444-bec3-d67d474f43ee)

**Comment by larskka** [comment]
Yes it would be great, but im not sure if the wirring is correct. Its for the elder medinav, i have seen som sellers write , that it wont support r-link?

**Comment by matzel82** [comment]
You need a diagram for the connection ? I can give you… I need your VIN

the lead at the picture is for MediaNav UlC4 not for RLink. It’s different.

**Comment by matzel82** [comment]
I think that’s important to config the camera at RLink. To „yes“

**Comment by larskka** [comment]
Vin nr.  vf1rfa00063460938 thankyou

**Comment by larskka** [comment]
> You need a diagram for the connection ? I can give you… I need your VIN
> 
> the lead at the picture is for MediaNav UlC4 not for RLink. It’s different.

vf1rfa00063460938 thank you :-)

**Comment by matzel82** [comment]
Apropos… you have to visit a garage, there is a OTS to reprogramming your ECUs

**Comment by matzel82** [comment]
Im not shure if you have the correct connection. Normally you use the connection from „Radio“ behind the Clovebox. There I can’t find so long the diagram. But I have attached you the manual to install the camera system originally. There are the correct instructions an a diagram to connect. I hope it’s helpful.
[EBA8201672189.PDF](https://github.com/cedricp/ddt4all/files/14418671/EBA8201672189.PDF)

**Comment by larskka** [comment]
Thank you so much :-) my fault i have trying to connect on the back of  r-link screen an not behandling the clovebox. So i learned a lesseon today

**Comment by AndyIreland** [comment]
so did you have to join your camera wires to the 32pin connector then?

**Comment by larskka** [comment]
![20240227_211339](https://github.com/cedricp/ddt4all/assets/161332104/5b12086b-2217-45ae-9212-2b38e2388788)
Now i found a 32 pin over clovebox.  But there is cable in the pins i think i should use Brown, yellow, red and black . Is it the right cable? If shall i just cut the cable and add the ones from camera? And finale page 22 in the instruktion  what Are the cables A? Are you lookind in the plug from the behind where you plug in the cable?  Is 21 video + 22 video -  23 supply + 24 supply - ? Thank you for your time:-)
